### ISSUE TITLE
Convergence issues encountered when using CMatch.

### ISSUE THREAD

**Issue by suuuuju** [microsoft/NeuralSpeech#104]
Currently, I can basically reproduce the results on the original data. Because I want to use the mmd and adv methods inside, I switched to a cross-accent dataset and set the vocabulary to 1000. However, it has never converged. Even if I adjust the learning rate very small, the loss is still high. After changing the vocabulary size to 30, the loss can converge but the word error rate is very high. So I replaced the encoder with wav2vec2 encoder pre-trained on 960 hours of librispeech. I connected it with a linear layer at the back to keep the same dimension as before, and everything else remained unchanged. But it still cannot converge. Even if I keep reducing the learning rate, the loss will not decrease. I would like to ask where should I start troubleshooting?

**Comment by jindongwang** [comment]
Can you adjust the tradeoff parameter of CMatch to better balance the MMD term?
